### DISCUSSION TITLE
Property 'getData' does not exist on type '{ tagName: string; getData?: ((el: HTMLElement) => HtmlBlockData<T>) | undefined; } | undefined'.

### DISCUSSION THREAD

**Discussion by RandyLiu6410** [react-page/react-page#1074] [Q&A]
When I use @react-page/plugins-slate, I met this problem when building. Is this a bug of @react-page/plugins-slate?

The version of packages I use are
"@react-page/editor": "^3.2.1",
"@react-page/plugins-background": "^3.2.1",
"@react-page/plugins-divider": "^3.2.1",
"@react-page/plugins-html5-video": "^3.2.1",
"@react-page/plugins-image": "^3.2.1",
"@react-page/plugins-slate": "^3.2.1",
"@react-page/plugins-spacer": "^3.2.1",
"@react-page/plugins-video": "^3.2.1",

**Comment by macrozone** [comment]
@RandyLiu6410 maybe its a strict null check? I did not activate strict null check yet on the project (due to legacy reasons), but i plan to do so.

Can you disable strict null check for the time beeing and check again. And can you provide further information, if you have some?
